plugins/woocommerce-admin/package.json

```diff
@@ -32,8 +35,7 @@
 		"packages:fix:textdomain": "node ./bin/package-update-textdomain.js",
 		"packages:watch": "cross-env WC_ADMIN_PHASE=development pnpm run:packages -- start --parallel",
 		"run:packages": "pnpm run --filter ../../packages/js/",
-		"start": "cross-env WC_ADMIN_PHASE=development pnpm -w exec turbo run build --filter='./packages/js/*' && cross-env WC_ADMIN_PHASE=development pnpm run build:feature-config && concurrently \"cross-env WC_ADMIN_PHASE=development webpack --watch\" \"cross-env WC_ADMIN_PHASE=development pnpm packages:watch\"",
-		"test": "node ./node_modules/require-turbo && pnpm run test:client",
+		"start": "cross-env WC_ADMIN_PHASE=development pnpm -w run build --filter='./packages/js/*' && cross-env WC_ADMIN_PHASE=development pnpm run build:feature-config && concurrently \"cross-env WC_ADMIN_PHASE=development webpack --watch\" \"cross-env WC_ADMIN_PHASE=development pnpm packages:watch\"",
```

Comment:
If we make the changes to `"dev"`, could we change this to `pnpm run dev -- --watch && pnpm packages:watch` or something? Most of this is a duplicate from above, so, it would just be great if we could consolidate it here.

Reply:
This was potentially hit by find/replace. I think I'd rather not change any internal scripts as part of this PR unless necessary. I think its still ok for these to call turbo right